### PR TITLE
fixed centos & fedora support

### PR DESCRIPTION
**Pull request by bh90210**
#174 #154

### CODE REVIEW

**Comment by bh90210** [comment]
also dropped redhat support for now once we figure a way to test on will reinstate :+1:

File: README.md [hunk 1]
```diff
@@ -48,17 +48,19 @@ Make sure you have the xcode command line tools installed. This can be done by r
 
 #### Ubuntu 18.04, Debian 9, Zorin 15
 
-`sudo apt install pkg-config build-essential libgtk-3-dev libwebkit2gtk-4.0-dev`
+`sudo apt install libgtk-3-dev libwebkit2gtk-4.0-dev`
```
Comment (leaanthony) [inline README.md]: pkg-config and build-essential are needed for gcc - at least in Ubuntu 16.04. Has that changed?

**Comment by leaanthony** [comment]
At some point we will need to add in distro version checking, but am happy with this PR! Good job! 👍

**Comment by bh90210** [comment]
absolutely! :+1:

File: README.md (hunk 1 above)
Comment (bh90210) [reply README.md]: nope, it is 100% needed. I removed it for consistency. For the rest of the distros we only suggest webkit libs.
minor thing. but can be potentially confusing. I have no strong opinion this was just a suggestion. proceed however you feel.

**Comment by leaanthony** [comment]
Go home Cloudflare Pages - you're drunk!

**Pull request merged**